frontend/src/footer/footer-template.hbs

```diff
@@ -1,5 +1,5 @@
 <figure class="footer-dh">
-    <p>Developed by</p>
+    <p>{{#i18n 'footer.developed-by'}}Developed by{{/i18n}}</p>
     <a href="https://dig.hum.uu.nl" target="_blank"><img class="dhlab"
             src="{{static 'image/dighum-logo-blue.svg'}}"></a>
```

Comment:
Should we already proactively change the Footer to only include the UU logo and DHLab name?